tests/docker/Makefile

```diff
@@ -22,7 +22,7 @@ lint: build
 	docker run -t -u $(uid):$(gid) -v $(top_srcdir):/source --tmpfs /var/lib/postgresql $(name) tox -e pylint
 
 shell:
-	docker run -ti --rm -u $(uid):$(gid) -v $(top_srcdir):/source --tmpfs /var/lib/postgresql $(name) /bin/bash
+	docker run -ti --rm --ulimit core=-1 -u $(uid):$(gid) -v $(top_srcdir):/source --tmpfs /var/lib/postgresql $(name) /bin/bash
```

Comment:
What does "--ulimit core=-1" do?

Reply:
As the commit says: It enabled the container to produce core dumps.